commit/merkleroot/outcome.go

```diff
@@ -188,7 +188,9 @@ func buildReport(
 				ChainSel:     root.ChainSel,
 				SeqNumsRange: root.SeqNumsRange,
 				MerkleRoot:   root.MerkleRoot,
-				// TODO: add OnRamp as a string?
+				// NOTE: this hex encoding is just for the sake of giving the onramp address
+				// a value in this key struct.
+				OnRampAddress: hex.EncodeToString(root.OnRampAddress),
```

Comment:
Shouldn't this use `typeconv.AddressBytesToString)`?

Reply:
This is just to make the onramp address comparable in this scope of code, its not used anywhere else. Technically we should be able to do `string(root.OnRampAddress)` to make that clear